manifests/b/buchen/portfolio/0.69.1/buchen.portfolio.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: buchen.portfolio
+PackageVersion: 0.69.1
```

```suggestion
PackageVersion: 0.69.1
Scope: machine
```

Metadata for better package correlation